source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -74,11 +74,12 @@ namespace RendererQml
 		context->setCardRootElement(uiCard);
 		uiCard->Property("readonly property int margins", std::to_string(margin));
         uiCard->AddFunctions("signal buttonClicked(var title, var type, var data)");
-        uiCard->Property("implicitHeight", "adaptiveCardLayout.implicitHeight");
+		//1px extra height to accomodate the border of a showCard if present at the bottom
+        uiCard->Property("implicitHeight", "adaptiveCardLayout.implicitHeight+1");
 		uiCard->Property("Layout.fillWidth", "true");
 		uiCard->Property("readonly property string bgColor", context->GetRGBColor(context->GetConfig()->GetContainerStyles().defaultPalette.backgroundColor));
         uiCard->Property("color", "bgColor");
-        uiCard->Property("border.color", context->GetColor(AdaptiveCards::ForegroundColor::Default, false, false));
+		uiCard->Property("border.color", isChildCard? " bgColor" : "'#B2B2B2'");
```

Comment:
Is there no way to set this hardcoded color in the config?

Reply:
The host config mainly has the values only for the choices of any card property (like for color its the seven choices default, dark, light, accent, etc). We may have to introduce another json or css file to store all the colors which are not adaptive card properties.